src/sentry/static/sentry/app/utils/streamManager.jsx

```diff
@@ -24,6 +24,8 @@ class StreamManager {
     items = [].concat(items);
     if (items.length === 0) return this;
 
+    items = items.filter((item) => item.hasOwnProperty("id"));
```

Comment:
Without the filter `push` would add `undefined` to the list if the item did not include an `id` property.